qdrant-landing/content/documentation/guides/running-with-GPU.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Running with GPU
+weight: 10
+---
+
+# Running with GPU
+
+Starting from version v1.13.0, Qdrant offers support for GPU acceleration. 
+
+GPU support is not included into the default Qdrant binary, as it requires additional dependencies and libraries. Instead, we provide separate Docker images with GPU support.
+
+
+## Configuration
+
+Qdrant includes a number of configuration options to control GPU usage. The following options are available:
+
+```yaml
+gpu:
+    # Enable GPU indexing.
+    indexing: false
+    # Force half precision for `f32` values while indexing.
+    # `f16` conversion will take place 
+    # only inside GPU memory and won't affect storage type.
+    force_half_precision: false
+    # Used vulkan "groups" of GPU. 
+    # In other words, how many parallel points can be indexed by GPU.
+    # Optimal value might depend on the GPU model.
+    # Proportional, but doesn't necessary equal
+    # to the physical number of warps.
+    # Do not change this value unless you know what you are doing.
+    # Default: 512
+    groups_count: 512
+    # Filter for GPU devices by hardware name. Case insensitive.
+    # Comma-separated list of substrings to match 
+    # against the gpu device name.
+    # Example: "nvidia"
+    # Default: "" - all devices are accepted.
+    device_filter: ""
+    # List of explicit GPU devices to use.
+    # If host has multiple GPUs, this option allows to select specific devices
+    # by their index in the list of found devices.
+    # If `device_filter` is set, indexes are applied after filtering.
+    # By default, all devices are accepted.
+    devices: null
+    # How many parallel indexing processes are allowed to run.
+    # Default: 1
+    parallel_indexes: 1
+    # Allow to use integrated GPUs.
+    # Default: false
+    allow_integrated: false
+    # Allow to use emulated GPUs like LLVMpipe. Useful for CI.
+    # Default: false
+    allow_emulated: false
+```
+
+It is not recommended to change these options unless you are familiar with the Qdrant internals and the Vulkan API.
+
+
+## Standalone GPU Support
+
+For standalone usage, you can build Qdrant with GPU support by running the following command:
+
+```bash
+cargo build --release --features gpu
+```
+
+Ensure your device supports Vulkan API v1.3. This enables compatibility with Apple Silicon, Intel GPUs, and CPU emulators.
```

Comment:
I'm not sure if it makes sense to mention that the device should support some device features like `f16`, `u8` operations ect. All actual server grade gpus support everything. From all tested devices I got only old laptop nvidia MX150 GPU with does not support some features